pkg/containerwatcher/v1/container_watcher.go

```diff
@@ -225,10 +228,6 @@ func CreateIGContainerWatcher(cfg config.Config, applicationProfileManager appli
 	// Create a dns worker pool
 	dnsWorkerPool, err := ants.NewPoolWithFunc(dnsWorkerPoolSize, func(i interface{}) {
 		event := i.(tracerdnstype.Event)
-		// ignore events with empty container name
```

Comment:
We will not ignore events with empty containers because they are needed by the DNS manager.
Moved this logic to the rule manager since we must have container name there.